requirements.txt

```diff
@@ -20,3 +20,10 @@ pdoc3
 pypubsub
 bleak
 packaging
+mypy
+mypy-protobuf
+types-protobuf
+types-tabulate
+types-requests
+types-setuptools
+types-PyYAML
```

Comment:
Probably self-explanatory, but this adds mypy, the mypy-protobuf plugin that generates the `.pyi` files, and type stubs for what existed.